W-15924012-note-Tested-Devices (#741)
mulesoft/docs-runtime-manager: cloudhub/modules/ROOT/pages/vpn-about.adoc

* 4.30 or later for dynamically routed Anypoint VPN connections
|===

> If there is not a specific *Device Configuration* file downloadable from the *Get VPN Config* menu for any of the listed devices, use the *Generic* VPN configuration file.

Reviewer: ```suggestion
If there is no specific *Device Configuration* file downloadable from the *Get VPN Config* menu for any of the listed devices, use the *Generic* VPN configuration file.
```

> image::vpn-device-generic-config.png[Generic VPN configuration file.]

Reviewer: ```suggestion
image::vpn-device-generic-config.png[Generic VPN configuration file option]
```
